### ISSUE TITLE
Function AAdd() doesn't support third parameter (Xbase++ dialect)

### ISSUE THREAD

**Issue by DenGhostYY** [X-Sharp/XSharpPublic#1287]
**Describe the bug**
Function [AAdd()](https://www.xsharp.eu/runtimehelp/html/M_XSharp_RT_Functions_AAdd.htm) doesn't support third parameter.

**Example 1. To Reproduce .prg**
```
procedure Main()
    local arr := {"a", "b", "c"}

    AAdd(arr, "d", 2)
    ? arr

    return
```

**Expected behavior (xBase++ exe)**
Output
```
{a, d, b, c}
```

**Actual behavior (X# compiler)**
```
error XS1501: No overload for method 'XSharp.RT.Functions.AAdd(array, usual)' takes 3 arguments
```

**Additional context**
X# Compiler version 2.16.0.5 (public)
-dialect:xBase++ -xpp1 -lb -memvar -vo1 -vo3 -vo5 -vo10 -vo15 -vo16

Xbase++ [AAdd()](https://doc.alaska-software.com/content/fdc_xppfcref_aadd.html)

**Comment by cpyrgas** [comment]
Confirmed done

**Comment by RobertvanderHulst** [comment]
This is included in the beta for 2.17. I have granted you @DenGhostYY access to the beta area on the X# website.
